### PR TITLE
attributes update for netcdf

### PR DESCRIPTION
**Pull request by JonathonMSmith**
# Description

Addresses #5 

Added basic unit attribute data for output variables.
Modified model objects to_netcdf function to convert model_object.MetaData to an attributes dictionary for netcdf export.

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- pytest locally
- manually saved and loaded netcdf with netCDF4 and xarray to inspect meta data

**Test Configuration**:
MacOS 10.15.7
Python 3.8.8
* Conda Environment

# Checklist:

- [X] Make sure you are merging into the ``develop`` (not ``main``) branch
- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules
- [X] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: sami2py/_core_class.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def __init__(self, tag, lon, year, day, outn=False, test=False):
         deni : (4D ndarray)
             Ion density by species (cm^-3)
```
Comment (jklenzing) [inline sami2py/_core_class.py:69]:
```suggestion
            Ion density for each ion species (cm^-3)
```

**Review by jklenzing — CHANGES_REQUESTED** [review]
This is looking good.  Code works as expected, generated netcdf has all expected attrs.

Could you add 'long_name' to the data attrs?This will help the code be slightly more self-descriptive if someone looks at the files.  You can use the descriptions of the variables from the docstrings (see suggestion below).

**Comment by jklenzing** [comment]
Comparing the 3.9 ubuntu pytest report to the develop branch, I see no new warnings.

File: sami2py/_core_class.py (hunk 1 above)
Comment (JonathonMSmith) [reply sami2py/_core_class.py:69]: Sounds good, should I just go ahead and make sure all of the SPASE/PSL netCDF standard attrs are there?

Comment (jklenzing) [reply sami2py/_core_class.py:69]: I'll leave it up to you depending on how much work that takes.

**Review by jklenzing — APPROVED** [review]

**Pull request merged**